tests/ui/llm_studio_page.py

```diff
@@ -0,0 +1,241 @@
+from hac_playwright.pages.base import BasePage
+from playwright.sync_api import expect
+
+CLOUD_FILESYSTEM_PATH = "/home/llmstudio/mount/data/user/"
```

Comment:
As discussed, this won't work for local tests.
Probably an env variable in the makefile together with an if-else condition will do the job (unless there's a more direct way of detecting running on the cloud):
```
test-ui-local: 
	$LOCAL_TEST=true \
        (PW_DEBUG) $(PIPENV) run pytest \ ...

```
Within the python code:
`if "LOCAL_TEST" in os.environ`

Reply:
I can create an env variable to store the local directory to run the tests locally. WDYT? eg:
`export FILESYSTEM_PATH=<path to local filesystem>`
you can manually export this env var to add path which shall trump `$CLOUD_FILESYSTEM_PATH`

Reply:
There may be other places where one may need to distinguish between local vs. cloud (e.g. disable login for local), having a flag may be more flexible.

Reply:
makes sense yess

Reply:
In general, Page Objects should not contain any test logic and test data, like specific paths, names etc.
It's just an abstraction layer to express your tests steps.
So all paths, names belong to test cases/fixtures only.

Reply:
@mnezh you asked me to remove filesystem path from the testfile